amun/dockerfile.py

```diff
@@ -137,4 +177,6 @@ def create_dockerfile(specification: dict) -> tuple:
     dockerfile += "USER 1042\n"
     dockerfile += "WORKDIR /home/amun"
 
+    dockerfile = _format_dockerfile(dockerfile)
```

Comment:
Not to break the current Amun deployment, let's create a condition (e.g. env variable THOTH_USE_ARGO`) which could be used also elsewhere until we turn the switch completely with Argo workflows.

Reply:
Good point, will do. Thanks!